spec/lib/api/v3/utilities/path_helper_spec.rb

```diff
@@ -31,8 +31,20 @@
 describe ::API::V3::Utilities::PathHelper do
   let(:helper) { Class.new.tap { |c| c.extend(::API::V3::Utilities::PathHelper) }.api_v3_paths }
 
-  shared_examples_for 'api v3 path' do
-    it { is_expected.to match(/^\/api\/v3/) }
+  shared_examples_for 'path' do |url|
+    it "provides the path" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.